docs/quickstart/aws.md

```diff
@@ -1,6 +1,6 @@
 ## Requirements 
 
-This quickstart assumes you have access to `AdministratorAccess` policy to AWS. 
+This quickstart assumes you have the ability to assign  `AdministratorAccess` policy to AWS. 
```

Comment:
Sorry for all the back and forth.. this should be "you are assigned `AdministratorAccess` right? Not necessarily the ability to assign the role...